draft-irtf-cfrg-frost.md

```diff
@@ -95,7 +95,7 @@ a signature.
 
 For select ciphersuites, the signatures produced by this draft are compatible with
 {{!RFC8032}}. However, unlike {{!RFC8032}}, signatures produced by FROST are not
-deterministic, since deriving nonces deterministically, is insecure in a multi-party
+deterministic, since deriving nonces deterministically is insecure in a multi-party
 signature setting.
```

Comment:
Unrelated to this PR, but this sentence in general is even a bit broad - deriving nonces deterministically is insecure for FROST, but not necessarily for any multi-party signature scheme.